resources/views/layouts/app.blade.php

```diff
@@ -20,7 +20,7 @@
     <meta name="robots" content="@yield('robots', Rapidez::config('design/search_engine_robots/default_robots', 'INDEX,FOLLOW'))"/>
     <link rel="canonical" href="@yield('canonical', url()->current())" />
 
-    @vite(['resources/css/app.css', 'resources/js/app.js'])
+    @vite(['resources/css/app.css', 'resources/js/app.js', 'vendor/rapidez/core/resources/js/turbolinks.js'])
```

Comment:
Can't we just eager load it so we don't have to specify it here?

Reply:
I had done it this way in the previous PR because not doing so caused issues.
However i cannot reproduce that issue anymore so my further work has probably fixed that.

I'll update it and just eager load it

Reply:
Unfortunately i have run into the issue again, the order of event listeners are incorrect so Vue cannot prevent the turbo navigation